snakemake/utils.py

```diff
@@ -111,33 +111,67 @@ def set_defaults(validator, properties, instance, schema):
     if not isinstance(data, dict):
         try:
             import pandas as pd
+            import polars as pl
```

Comment:
this should work independently for pandas and polars. Otherwise, people will always have to have installed both just to validate one of them.

Reply:
But both are already part of `snakemake` dependencies, no?